### DISCUSSION TITLE
Cant get Auth token anymore?

### DISCUSSION THREAD

**Discussion by Sigoth** [victornpb/undiscord#316] [1. Questions & Answers] [answered]
Was working absolutely fine until about 3-4 days ago, now when i go to delete i can get Author, Guild/channel ect. But everytime i click Authorization it doesnt populate, tested this on mutiple accounts, reinstalled ect. Anyone else having this issue? If so anyone know how to fix?

I cant even manually find my auth token using methods online, seems like going into console ect doesnt work anymore, any help appreciated!

**Comment by jribbink** [comment]
It looks like it is because V6 is deprecated.  Someone made a PR for this repo and it seems to work for me

https://github.com/abbydiode/UndiscordPlus

**Comment by victornpb** [comment, marked as answer]
It has been fixed, please make sure you're running an updated version of Undiscord.
